### DISCUSSION TITLE
Introduction

### DISCUSSION THREAD

**Discussion by deena101** [cooperrc/computational-mechanics#18] [General]
Hi! My name is Adeena, I am a rising senior with a concentration in Design and Manufacturing. A fun-fact about me is that I have a disappearing birthmark on my thumb (I had it when I was younger, it disappeared through middle - high school and now it's back). I am taking this class so that I can stay on track to graduate on time. This class will also help me in my career as most companies require you to have some knowledge in coding and I plan on creating a device to help the visually impaired for my Master's and I will need coding for that project.

**Comment by tempcooper** [comment]
> creating a device to help the visually impaired for my Master's and I will need coding for that project.

So cool! Are you transforming/amplifying image information somehow? or using some kind of image recognition->interpretation?

**Reply by deena101** [reply]
I plan on making an image based/sensory gps system or something along those lines. Still have a few kinks to figure out.
